Command/AbstractCommand.php

```diff
@@ -96,7 +96,7 @@ private function doDump(AssetInterface $asset, OutputInterface $stdout)
                 }
             }
 
-            $stdout->writeln(sprintf(
+            $stdout->text(sprintf(
```

Comment:
the typehint of the method should be changed. these methods are not part of ``OutputInterface``